relay-general/src/store/transactions/processor.rs

```diff
@@ -29,13 +30,26 @@ pub struct TransactionNameConfig<'r> {
 #[derive(Default)]
 pub struct TransactionsProcessor<'r> {
     name_config: TransactionNameConfig<'r>,
+    span_desc_rules: Option<Vec<SpanDescriptionRule>>,
     scrub_span_descriptions: bool,
 }
 
 impl<'r> TransactionsProcessor<'r> {
     pub fn new(name_config: TransactionNameConfig<'r>, scrub_span_descriptions: bool) -> Self {
+        let mut span_desc_rules = None;
+        if scrub_span_descriptions && !name_config.rules.is_empty() {
```

Comment:
I tried something like:
```rust
let span_desc_rules = if ... {
  // generate rules
} else {
  None
}
```
But could not deal with lifetimes. I think this is fine for now.